src/styles/blocks/stars0.css

```diff
@@ -0,0 +1,32 @@
+.stars--0 {
+  display: flex;
+  justify-content: space-between;
+  width: 96px;
+  margin-bottom: 4px;
+}
+
+.stars--0 div {
+  background-size: cover;
+  height: 16px;
+  width: 16px;
+}
+
+.stars--0 div:nth-child(1) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--0 div:nth-child(2) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--0 div:nth-child(3) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--0 div:nth-child(4) {
+  background-image: url(../../images/star.svg);
+}
+
+.stars--0 div:nth-child(5) {
+  background-image: url(../../images/star.svg);
```

Comment:
The use of `nth-child` selectors for applying `background-image` is not optimal for this task. The task requires using pseudo-selectors for styling stars. Consider using `::before` or `::after` pseudo-elements to apply the background images.